### PR TITLE
Adding "Test Connection" for LDAPAuthenticationEntity.

### PR DESCRIPTION
**Pull request by akhil-jha**
Signed-off-by: Akhil Jha

### CODE REVIEW

File: airgun/entities/ldap_authentication.py [hunk 1]
```diff
@@ -53,6 +53,13 @@ def delete(self, entity_name):
         view.flash.assert_no_error()
         view.flash.dismiss()
 
+    def testConnection(self, values):
```
Comment (omkarkhatavkar) [inline airgun/entities/ldap_authentication.py:56]: keep the function name lowercase as test_connection or check_connection

Comment (akhil-jha) [reply airgun/entities/ldap_authentication.py]: Requested changes made.

**Comment by omkarkhatavkar** [comment]
> LGTM but Omkar's comment needs to be addressed.
> Just a suggestion, maybe generalize this to also support testing a connection in an already created LDAP entity?

General idea was to test the connection for all auth sources to see if connection work this kind of smoke test first all LDAP entities. We can definitely add in create entity but idea was to test all connections not seeing an error message after click on the button. We had some customer cases saying they were not able to connect. https://bugzilla.redhat.com/show_bug.cgi?id=1829551

**Comment by omkarkhatavkar** [comment]
@akhil-jha please post the test result

**Comment by akhil-jha** [comment]
`#pytest tests/foreman/ui/test_ldap_authentication.py -k test_positive_test_connection_functionality

================================ test session starts ============================
platform linux -- Python 3.6.8, pytest-4.6.3, py-1.8.1, pluggy-0.13.1
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/akjha/satelliteqe/robottelo
plugins: cov-2.8.1, forked-1.1.3, services-1.3.1, mock-1.10.4, xdist-1.32.0
collecting ... 2020-05-22 10:40:36 - conftest - DEBUG - Collected 22 test cases
2020-05-22 16:10:42 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f08f8ed6080
2020-05-22 16:10:42 - robottelo.ssh - INFO - Connected to [dhcp-3-70.vms.sat.rdu2.redhat.com]
2020-05-22 16:10:42 - robottelo.ssh - INFO - >>> rpm -q satellite
2020-05-22 16:10:44 - robottelo.ssh - INFO - <<< stdout
satellite-6.8.0-0.3.beta.el7sat.noarch

2020-05-22 16:10:44 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f08f8ed6080
2020-05-22 16:10:44 - robottelo.host_info - DEBUG - Host Satellite version: 6.8
2020-05-22 16:10:44 - robottelo.helpers - DEBUG - Generated file bz_cache.json with BZ collect data
collected 22 items / 21 deselected / 1 selected                                                                                                  

tests/foreman/ui/test_ldap_authentication.py .                                                                                          [100%]`

**Pull request merged**